docker/base-images/base-admin-tools.Dockerfile

```diff
@@ -9,10 +9,11 @@ RUN apk add --update --no-cache \
     mysql-client \
     postgresql-client \
     py-pip \
+    tini \
     && pip install cqlsh
 
 # set up nsswitch.conf for Go's "netgo" implementation
 # https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-424546457
-RUN test ! -e /etc/nsswitch.conf && echo 'hosts: files dns' > /etc/nsswitch.conf
+RUN test ! -e /etc/nsswitch.conf && echo 'hosts: files dns' > /etc/nsswitch.conf || true
```

Comment:
Any context why this change?

Reply:
If the `test` command returns false, then the whole `RUN` command evaluate to `false` and the docker build fails because of that.

I have no idea why this error actually happened on my local build and not to anyone before, but for sure, either that command should not be conditional or it should have the `|| true` at the end...

Reply:
```suggestion
RUN test -e /etc/nsswitch.conf || echo 'hosts: files dns' > /etc/nsswitch.conf
```